### DISCUSSION TITLE
the upload button in "Upload proof of your academic status" is refreshing the page, and throwing an error message

### DISCUSSION THREAD

**Discussion by shadowboxe** [community/community#31850] [GitHub Education]
Hello, I've been trying to upload a picture of my college card, but the moment I hit upload, the page refreshed. An error message popped up, telling me to upload the image (as if I didn't), I also tried to take a picture directly, but it just keeps spinning; I think there's something wrong with the website.

**Comment by ihamidch** [comment]
I am also facing the same issue

**Reply by ghost** [reply]
did you manage to solve it?

**Reply by mr-september** [reply]
Pop the photo into Photoshop, play with Unsharp Mask, Auto Color/Contrast, save (under 1 Mb), and it finally went through.

**Comment by reFresherr0** [comment]
Same issue

**Reply by ghost** [reply]
same problem here, the GitHub support team should really resolve this.

**Comment by ihamidch** [comment]
> Hello, I've been trying to upload a picture of my college card, but the moment I hit upload, the page refreshed. An error message popped up, telling me to upload the image (as if I didn't), I also tried to take a picture directly, but it just keeps spinning; I think there's something wrong with the website.

I have a solution I use Take photo option and take a picture or taker student card in your hand and take photo of your student card 
This solution is much better My GitHub Student Pack is approved by this method

**Comment by reFresherr0** [comment]
I used (take photo) on my phone but only the front facing camera is working😆 it worked so whatever.

**Comment by GeoGrammenidis** [comment]
I have the same issue. I have tried to upload an image and it says "File is too small". As well as I have tried to take a photo with my phone but it seems that is bugged as well.

**Comment by BassemAdam** [comment]
i have the same issue

**Comment by werwer748** [comment]
not yet I have the same issue

**Comment by ixnoahlive** [comment]
did reproduce

**Comment by gabrielGAGRA** [comment]
This is almost 2 years old and the problem is still there

**Comment by SachaRG** [comment]
I am also having this issue...

**Comment by mahirsn** [comment]
I think we experienced something like this because our photos were larger than 1 MB. Select the "take it with the camera" option on the website from your phone. I solved it this way.

**Reply by mishmanners** [reply]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:

- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB

**Comment by gabrielGAGRA** [comment]
You are right. It’s this tiny detail that instead of throwing an error to the user it just denies the process request and outputs nothing.
My sincere thanks to you and my sincere screw you to GitHub for this terrible code

**Comment by mishmanners** [comment]
Yes, photos need to be small, otherwise there will be an error. We apologise for this, and glad most of you were able to work this out. Please also ensure you:
- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB

**Comment by rmpasswd** [comment]
In my case, My picture was .jpg format, 800Kb. But the pixel resolution was too much.
**Github did not mention that there is a max limit, they only mention that minimum is 1024x768**. Mine was 5 times more than that. Cuz you know how smartphones these days are stuffing more pixels...
Changing format from .jpg to .png increased the file size to ~8MB 
I solved it by using 'resize' feature of good old 'mspaint', 50% reduction and then it worked.
